src/sentry/static/sentry/app/views/eventsV2/eventView.tsx

```diff
@@ -787,6 +787,10 @@ class EventView {
 
     const picked = pickRelevantLocationQueryStrings(location);
 
+    // normalize datetime selection
+
+    // picked = omit(picked, ['start', 'end', 'period', 'statsPeriod', 'utc']);
```

Comment:
Could remove this.

Reply:
@markstory already removed in https://github.com/getsentry/sentry/pull/15315/commits/fc2972525b73f43a2cd883a65300849444add236